### DISCUSSION TITLE
Mid-Latitude incoming solar, ET & Clouds

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/cesm_dev#27] [2. Model bias]
In an attempt to improve RESTOM @adamrher found high biases in incoming shortwave over mid-latitudes of NH land in summer.

![image](https://github.com/user-attachments/assets/96f3aab8-f4e4-4098-9bff-f84c0ec66b0a)

We do have some suggestions from [a coupled model PPE](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2024GL108372) by @czarakas and friends suggests that land model parameters can influence incoming shortwave and clouds and are kicking off a slew of F-cases to investigate these in [LWMG_dev](https://github.com/NCAR/LMWG_dev/issues).

**Comment by wwieder** [comment]
These experiments are more of a sensitivity test, to see the extent to which land parameters can influence, but informed by [plots here](https://github.com/NCAR/LMWG_dev/discussions/75#discussioncomment-11479094) from @aswann, @slevis-lmwg and I chose the following land model parameter changes in increase ET and decrease FSDS, especially in summer:
1. Control CTSM5.3 parameters in an FLT case + the following individual perturbations.
2. **d_max** (soil hydrology parameter, low values from PPE chosen = 10)
3. **uplim_destruct_metamorph** (snow parameter, high values from PPE = 250)
4. **jmaxb0** (photosynthesis parameter, high value from PPE = 0.05)

**Reply by slevis-lmwg** [reply]
https://github.com/NCAR/LMWG_dev/issues/78
https://github.com/NCAR/LMWG_dev/issues/79
https://github.com/NCAR/LMWG_dev/issues/80
https://github.com/NCAR/LMWG_dev/issues/81

**Comment by wwieder** [comment]
I'll also note here that we have some indication that model and parameter changes that came in with CTSM5.3 in B-112, #15, may also be responsible for lower ET and clouds that increase FSDS, here for JJA? 
- FSDS changes are not as widespread at CIRES results (above) may suggest.  
- Differences not necessarily statistically significant
- See also #26 and diags between [B-112 vs B-111](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/setsIndex.html)

<p align="center"> <img src="https://github.com/user-attachments/assets/17b21ac7-1584-4f48-9ff4-c0019a4bbc5a" width="450" /></p>

**Comment by wwieder** [comment]
We are able to decrease summer incoming solar radiation in the NH.  Changing photosynthetic parameter alone is the largest lever from the list above, and doing all three changes at once has (slightly) larger combined effects.

It's helpful to flip between plots showing results from [all three parameters](https://webext.cgd.ucar.edu/FLT2000/f.e30_alpha04a.FLT2000BgcCrop.j5d10u250.lmwg_dev_87/lnd/f.e30_alpha04a.FLT2000BgcCrop.j5d10u250.lmwg_dev_87.11-20-f.e30_alpha04a.FLT2000BgcCrop.CTRL.lmwg_dev_78.11-20/set2/set2_JJA_FSDS.gif) (also belowa) vs. [just jmaxb0](https://webext.cgd.ucar.edu/FLT2000/f.e30_alpha04a.FLT2000BgcCrop.jmaxb0_005.lmwg_dev_84/lnd/f.e30_alpha04a.FLT2000BgcCrop.jmaxb0_005.lmwg_dev_84.11-20-f.e30_alpha04a.FLT2000BgcCrop.CTRL.lmwg_dev_78.11-20/set2/set2_JJA_FSDS.gif).  Seems like differences from combined land parameter changes are ~12 W/m2, whereas @adamrher's bias plots above are > 30 W/m2

![image](https://github.com/user-attachments/assets/cdaea558-1748-4616-bcc9-6ac468f8966d)

Thanks to @slevis-lmwg for doing these runs and diagnostics.

**Reply by adamrher** [reply]
Awesome work. I'd like to run some transient F-cases with both these configurations (just maxb0 vs. all three; could you provide the namelist settings?) so I can evaluate them against obs. Are these done in SP mode, or do I need to run the F-cases w/ BGC?

**Reply by slevis-lmwg** [reply]
@adamrher mine are in BGC mode. Details here:
https://github.com/NCAR/LMWG_dev/issues/84
https://github.com/NCAR/LMWG_dev/issues/87

jmaxb0 is a photosynthesis parameter, and it would likely have a similar effect in SP mode, but you would miss any feedback due to changes in Leaf Area Index.

Let me know if you would like me to check your cases before you submit them.

**Reply by adamrher** [reply]
OK, thanks for the info. We'll chat about whether to do SP or BGC (or both) at tmrw's CAM7 mtg.

**Comment by wwieder** [comment]
Some encouraging results in [LMWG_dev#86](https://github.com/NCAR/LMWG_dev/issues/86). Full diagnostics are coming but this run by @slevis-lmwg that clones #25 with a new parameter set from @linniahawkins seems to decrease incoming solar (FSDS) and lower temperatures (TSA), shown below for annual means (averaged over years 11-20).

![image](https://github.com/user-attachments/assets/3d04180e-c8ab-4f5a-8356-6ac8080b7a07)

![image](https://github.com/user-attachments/assets/e80b5520-ff8a-4dc7-8f67-9ec9394f37b7)

**Comment by wwieder** [comment]
There's been lots of email and discussion on this, but I wanted to put this figure from @adamrher up here, as he's trying to tune clouds in CAM7 to reduce incoming SW biases we're seeing in CESM3 (top right) vs. CESM2 (top left). Several tuning iterations are here from, with some variant of atune2 (middle right) seemingly prefered.

<img width="1254" height="1136" alt="image" src="https://github.com/user-attachments/assets/c01d9908-35aa-4535-a280-9337aa83ff52" />

Interestingly, @rosiealice noted that the NorESM team is seeing similar issues, both here and in the tropics #26, and are interested in tracking progress made here.

**Reply by adamrher** [reply]
This shows more clearly the impact of the tuning, subtracting the cesm3 cntl from the experiments:

<img width="3037" height="1885" alt="temp_dSWCF" src="https://github.com/user-attachments/assets/8f968411-8a96-469d-8d18-bf70f25c5eb5" />

**Comment by rosiealice** [comment]
Thanks @wwieder I think maybe our current FSDS biases are worse. e.g. from the current NorESM tag (beta04) FSDS vs CERES bias: 
<img width="3000" height="1000" alt="image" src="https://github.com/user-attachments/assets/53cd365e-331c-494c-b1bf-0b936dc4d862" />

We are doing a coupled model PPE right now and including 1) medlyn slope 2) d_max & dsl_init, 3) hydraulic failure mortality threshold and 4) z0mr as the land parameters, as well as peturbations from all other components. Hopefully this will give us some pointers to find a cloudier configuration! 

More coupled diagnostics here https://ns9560k.web.sigma2.no/datalake/diagnostics/noresm/tomast/n1850.ne30_tn14.noresm3_0_beta04-run1_yr001.20251103/XESMF_CLM-FATES/yr001-015/n1850.ne30_tn14.noresm3_0_beta04-run1_yr001.20251103/OBS_comparison/ANN/
